source/core/src/main/com/deco2800/game/components/story/basicdialogue/BasicDialogue.java

```diff
@@ -0,0 +1,78 @@
+package com.deco2800.game.components.story.basicdialogue;
+
+import com.deco2800.game.components.Component;
+import com.deco2800.game.components.dialoguebox.TextDialogueBox;
+import com.deco2800.game.components.story.StoryBase;
+import com.deco2800.game.entities.Entity;
+import com.deco2800.game.entities.factories.DialogueBoxFactory;
+import com.deco2800.game.services.ServiceLocator;
+
```

Comment:
short explanation on having a `BasicDialogue.java` - where does it come into play and why do you need it maybe.